src/tests/ftest/deployment/network_failure.py

```diff
@@ -0,0 +1,399 @@
+#!/usr/bin/python
```

Comment:
Since this is a cherry-pick, it's okay, but FYI that we should add the shebang to non-executable files

Reply:
I see. I used git cherry-pick, but I updated the commit message because the first commit message for master PR was outdated.

Reply:
Correction: we should NOT add the shebang to non-executable files